test.sh

```diff
@@ -61,32 +62,32 @@ then
 	fi
 
 	echo "> Creating images"
-	dd if=/dev/zero of=block count=10 bs=1M 2>&1 | sed 's/^/\t/g'
+	dd if=/dev/zero of=block count=10 bs=1M 2>&1 | sed 's/^/\t/'
 
 	echo "> Setting up the loop devices"
 	block="$(sudo losetup -f)"
 	echo "> Setting up block as $block"
-	sudo losetup $block block 2>&1 | sed 's/^/\t/g'
+	sudo losetup $block block 2>&1 | sed 's/^/\t/'
 
 	echo "> Formatting '$block' with passphrase '$passphrase'."
-	printf "%s" "$passphrase" | sudo cryptsetup luksFormat -q $block - 2>&1 | sed 's/^/\t/g'
+	printf "%s" "$passphrase" | sudo cryptsetup luksFormat -q $block - 2>&1 | sed 's/^/\t/'
 	echo "> Opening the device '$block' as /dev/mapper/temp-test"
-	printf "%s" "$passphrase" | sudo cryptsetup luksOpen -q $block temp-test - 2>&1 | sed 's/^/\t/g'
+	printf "%s" "$passphrase" | sudo cryptsetup open -q $block temp-test - 2>&1 | sed 's/^/\t/'
```

Comment:
`luksOpen` is replaced by `open --type luks`, not only by `open`.
The `global` `sed(1)` flag is needed for the multi-line outputs.